allauth/templates/account/email/email_confirmation_message.txt

```diff
@@ -1,8 +1,8 @@
-{% load account %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}Hello from {{ site_name }}!
+{% extends 'base.txt' %}{% load account %}{% block content %}{% user_display user as user_display %}{% load i18n %}{% autoescape off %}{% blocktrans with site_name=current_site.name site_domain=current_site.domain %}Hello from {{ site_name }}!
```

Comment:
Perhaps it would be better to have a `allauth/templates/account/email/base_message.txt` so that it is clear that this base is email message specific, and, it is scoped within the allauth template context.